llvm/test/CodeGen/AMDGPU/llvm.amdgcn.image.atomic.flt.ll

```diff
@@ -0,0 +1,40 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc -march=amdgcn -mcpu=gfx1200 -verify-machineinstrs < %s | FileCheck -check-prefix=GFX12 %s
```

Comment:
Should use an explicit -global-isel=0 for the DAG test; also don't really need -verify-machineinstrs